scripts/install.ps1

```
@@ -0,0 +1,617 @@
+    if ([string]::IsNullOrWhiteSpace($token)) {
+        return @{}
+    }
+    return @{ Authorization = "Bearer $token" }
+}
+function Resolve-LatestVersion {
+    $apiUrl = "https://api.github.com/repos/$Repo/releases/latest"
+    $apiMessage = ""
+    $rateLimited = $false
+    try {
+        $release = Invoke-RestMethod -Uri $apiUrl -Headers (Get-AuthHeaders) -UseBasicParsing
+        if ($release -ne $null -and -not [string]::IsNullOrWhiteSpace($release.tag_name)) {
```

PSScriptAnalyzer (1.25.0)

[warning] $null should be on the left side of equality comparisons.

Suggested fix: Use $null on the left hand side for safe comparison with $null.

(PSPossibleIncorrectComparisonWithNull)